inst/rmarkdown/templates/mortality/skeleton/skeleton.Rmd

```diff
@@ -1,120 +1,236 @@
 ---
 title: "Mortality survey"
-output: github_document
+output: word_document
 ---
 
-```{r setup, include=FALSE}
-knitr::opts_chunk$set(echo = TRUE)
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(echo = FALSE,
+                      warning = FALSE,
+                      error = TRUE)
 library(dplyr)
 library(knitr)
-library(epireports)
+library(sitrep)
 library(survey)
+library(ggplot2)
 ```
 
-```{r, message=FALSE}
-# study_data_raw <- readxl::read_excel("...")
-study_data_raw <- outbreaks::fluH7N9_china_2013
-
-study_data_cleaned <- study_data_raw %>%
+```{r read_data, message = FALSE}
+# use read_data stuff from outbreak?
```

Comment:
Yap - i think how you have it here is good. The outbreak template has grown a bit out of proportion and thats not necessary for survey right now. 
Also add an example section (commented out) on reading in the data dictionary from an excel?